[13.0][ADD] survey_crm_sale_generation: New module (#66)
OCA/survey: survey_crm_sale_generation/README.rst

.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
   !! This file is generated by oca-gen-addon-readme !!
   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsurvey-lightgray.png?logo=github
    :target: https://github.com/OCA/survey/tree/13.0/survey_crm_sale_generation
    :alt: OCA/survey
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/survey-13-0/survey-13-0-survey_crm_sale_generation
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/200/13.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

> This module allows link leads/opportunities and sales generated by

Reviewer: ```suggestion
This module allows to link the leads/opportunities with the quotations generated by
```

``survey_sale_generation`` and ``survey_crm_generation``.

**Table of contents**

.. contents::
   :local:

Configuration
=============

Configure your surveys as it's described in the instructions for
``survey_sale_generation`` and ``survey_crm_generation``

Usage
=====

> Once the surveys are submited a lead/opportunity (depending on the default options for

Reviewer: ```suggestion
Once the survey is submitted, a lead/opportunity (depending on the default value for
```
